server/src/pages/shotindex/controller.js

```diff
@@ -11,7 +11,8 @@ let firstRun = true;
 
 const queryParamModelPropertyMap = {
   q: "defaultSearch",
-  p: "pageNumber"
+  p: "pageNumber",
+  a: "accountId"
```

Comment:
Why do we need this?

Reply:
This is used to pass accountId stored in model to server.js  here https://github.com/mozilla-services/screenshots/pull/4759/commits/bbe67aa5e02520f07df14fc6832b4a6222e0e25e#diff-78f59c6794bdfdf3aa07ed2363aa584dR17

Reply:
Sure, that's where it is used.  But I don't think it's necessary.  The request to get the shots should have the account id in its cookies.

Reply:
got it, updated PR to remove passing accountId from queryParamModelPropertyMap